ansible/roles/slash/config/common/_slash_generator.py

```
@@ -0,0 +1,182 @@
+            )
+        )
+
+    return results
+
+
+def ensure_prompt_exists(prompt_root: Path, prompt_file: str) -> Path:
+    """Return the resolved path to a prompt file, ensuring it exists."""
+
+    prompt_path = (prompt_root / prompt_file).resolve()
+    if not prompt_path.is_file():
+        raise SlashGeneratorError(f"Prompt file not found: {prompt_path}")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)